tests/unit/amo/components/TestCollection.js

```diff
@@ -507,4 +474,30 @@ describe(__filename, () => {
     const wrapper = renderComponent();
     expect(wrapper.find('title')).toHaveLength(0);
   });
+
+  describe('errorHandler - extractId', () => {
+    it('returns a unique ID based on params', () => {
+      const props = {
```

Comment:
for maintainability, this should use the helper like `getProps({ params: { user: 'foo', ... } })`